src/form-elements/FormElementLocation.tsx

```diff
@@ -372,6 +372,7 @@ const LocationPicker = React.memo(function LocationPicker({
 
   const { isLoaded } = useJsApiLoader({
     googleMapsApiKey: googleMapsApiKey ?? '',
+    libraries: ['maps', 'marker', 'places'],
```

Comment:
Should we move this hook higher into the dom and make it a provider so it is only happening once?

Reply:
just load it in regardless? or check if there is a location/google element in the form?

Reply:
🤔 will it fail if you attempt to load a lib that the key does not support?

Reply:
No it won't on load, tested already.  It will only fail when you try and use it to do something it can't.

Reply:
mmm I think we iterate over the form elements to check if we need to load the script

Reply:
we'd have to do that each render in case of element lookups.

Reply:
Dun worry, Uncle Dave has an idea